Source/WebCore/accessibility/AccessibilityImageMapLink.h

```diff
@@ -76,7 +76,11 @@ class AccessibilityImageMapLink final : public AccessibilityMockObject {
     
 } // namespace WebCore
 
-SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::AccessibilityImageMapLink) \
-    static bool isType(const WebCore::AXCoreObject& object) { return object.isAccessibilityObject() && downcast<WebCore::AccessibilityObject>(object).isImageMapLink(); } \
-    static bool isType(const WebCore::AccessibilityObject& object) { return object.isImageMapLink(); } \
+SPECIALIZE_TYPE_TRAITS_BEGIN(WebCore::AccessibilityImageMapLink)
+    static bool isType(const WebCore::AXCoreObject& object)
+    {
+        auto* accessibilityObject = dynamicDowncast<WebCore::AccessibilityObject>(object);
+        return accessibilityObject && accessibilityObject->isImageMapLink();
+    }
+    static bool isType(const WebCore::AccessibilityObject& object) { return object.isImageMapLink(); }
```

Comment:
Should these lines be indented? I would've expected not, but maybe there's some pre-existing code style you're copying.

Reply:
They are indented for all (most?) classes as far as I can tell. See `git grep "    static bool isType" Source`.